Adds various tweaks for the Journalist

Rebruiza opened the pull request.

Rebruiza commented:
:woman_shrugging:

comma commented on maps/torch/job/outfits.dm:
```diff
@@ -675,7 +675,8 @@ Keeping them simple for now, just spawning with basic EC uniforms, and pretty mu
 	name = OUTFIT_JOB_NAME("Journalist - Torch")
-	backpack_contents = list(/obj/item/device/tvcamera = 1)
+	backpack_contents = list(/obj/item/device/tvcamera = 1,
+	/obj/item/clothing/accessory/badge/press = 1))
```
extra ) here

Rebruiza commented:
:clap: travis :clap: completed :clap: successfully :clap:

Rebruiza commented:
i'll squash later

Hubblenaut commented on code/modules/clothing/under/accessories/badges.dm:
```diff
@@ -6,26 +6,21 @@
 
 	if(!stored_name)
-		to_chat(user, "You polish your [src.name] fondly, shining up the surface.")
+		to_chat(user, "You write down your [src.name] and information on the [src], polishing the laminated surface.")
```
I believe the point of this was so the item pretends it already had the information on it. It doesn't make sense for most if not all subtypes of badges that the credentials are written on it by hand and without official verification.

A better way in general for these would have them automatically retrieve their holder's information. See tablet/laptop loadout.

Rebruiza replied:
:+1:

comma commented:
suqash nerd

comma commented on code/modules/clothing/under/accessories/badges.dm:
```diff
@@ -99,29 +96,50 @@
 	desc = "A leather-backed silver badge bearing the crest of the Defense Intelligence Agency."
 	icon_state = "diabadge"
-	slot_flags = SLOT_TIE
+	slot_flags = SLOT_BELT | SLOT_TIE
```
since slot_belt|tie is what base badge type does,you can just remove these lines entirely, they were only here to specifically overwrite it with tie-only.
Not seeing issue in letting people putting badges on belt for that 90s cop show look.

Rebruiza commented:
Squash SI BAd Shit MAn:clap: :+1:

Commit:
Adds various tweaks for the Journalist


?


Update outfits.dm
Update badges.dm
Update badges.dm
Update badges.dm
Update badges.dm
Update badges.dm
2017-12-01_PressStuff: Auto stash before merge of "2017-12-01_PressStuff" and "origin/2017-12-01_PressStuff"

ghost commented:
:point_right: this PR :point_left:
